guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -41,6 +41,11 @@ For more information, see {AdministeringDocURL}limiting-host-resources[Limiting
 
 [id="foreman-deprecations"]
 == Deprecations
+=== Preparing {Project} for operation in containerized environments
+- *`sendmail` is deprecated for delivering email* - Delivering email by calling the `sendmail` binary on the system is problematic in many cases.
+Therefore, it is deprecated and will be removed in a future version.
+As an alternative, use the SMTP mail delivery method.
+For more information, see link:https://docs.theforeman.org/3.16/Installing_Server/index-katello.html#Configuring_Server_for_Outgoing_Emails_foreman[Configuring Foreman server for outgoing emails].
```

Comment:
```suggestion
For more information, see {InstallingServerDocURL}Configuring_Server_for_Outgoing_Emails_foreman[Configuring {ProjectServer} for outgoing emails].
```

That way, the URL for `foreman` will point to Foreman; for `katello` to foreman+Katello.